### PR TITLE
Updated changes

### PR DESCRIPTION
**Pull request by Aquassaut**
Better words and sentences and descriptions and other things

### CODE REVIEW

File: source/includes/_changes.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Changes
```
Comment (jmorel) [inline source/includes/_changes.md]: Could you put Dataset Changes as a title to be in line with dataset metadata ?

File: source/includes/_changes.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+[
+    {...},
+    {
+        "change_uid": 126,
+        "dataset": {
+            "dataset_id": "changed_dataset",
+            "domain": "domain"
+        },
+        "user": {
+            "username": "delbingo"
```
Comment (jmorel) [inline source/includes/_changes.md]: not as neutral as we'd like :) Maybe johnsmith ?

File: source/includes/_changes.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+```
+
+> Example response
+
+```json
+[
+    {...},
+    {
+        "change_uid": 126,
+        "dataset": {
```
Comment (jmorel) [inline source/includes/_changes.md:85]: you didn't pass the expand parameter in the call, dataset should not be expanded.

File: source/includes/_changes.md (hunk 2 above)
Comment (jmorel) [reply source/includes/_changes.md]: You'd need to include the expand=user parameter in your call

File: source/includes/_changes.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+```
+
+Restore a dataset to the state it was in before the change (undoing the change).
+
+### Parameters
+
+> Example request
+
+```shell
+curl -XPUT https://yourdomain.opendatasoft.com/api/management/v2/restore_change/ \
```
Comment (jmorel) [inline source/includes/_changes.md:128]: missing space ? curl -X PUT ?

**Review by jmorel — CHANGES_REQUESTED** [review]

File: source/includes/_changes.md (hunk 3 above)
Comment (Aquassaut) [reply source/includes/_changes.md:85]: It's not expanded, the domain is part of the key. It's the other instance that is wrong :-)

File: source/includes/_changes.md (hunk 2 above)
Comment (Aquassaut) [reply source/includes/_changes.md]: Why ? In this instance, the user is not expanded.

File: source/includes/_changes.md (hunk 4 above)
Comment (Aquassaut) [reply source/includes/_changes.md:128]: There's no need for a space and people often don't put it. It is the same for most dash-parameters in UNIX-like utilities.

File: source/includes/_changes.md (hunk 1 above)
Comment (Aquassaut) [reply source/includes/_changes.md]: :+1:

File: source/includes/_changes.md (hunk 2 above)
Comment (Aquassaut) [reply source/includes/_changes.md]: Obvious :+1: for johnsmith, BTW

File: source/includes/_changes.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
+# Dataset Changes
+
+The changes of a dataset describe subsequent changes of states that affected the different sections of a dataset. Every action taken on any resource through POST, PUT or DELETE creates a change object that can be retrieved and acted upon.
+
+## The change object
+
+> An example change object
```
Comment (meganelacheny) [inline source/includes/_changes.md]: To ensure homogeneity in all examples, "An" should be deleted. Simply keep "Example change object".

File: source/includes/_changes.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+        },
+        "sections": [
+            "security"
+        ]
+    },
+    {...}
+]
+```
+
+## Restore a changes
```
Comment (meganelacheny) [inline source/includes/_changes.md]: Shouldn't it be "Restore a change" or "Restore changes"?

File: source/includes/_changes.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+
+```shell
+curl -XPUT https://yourdomain.opendatasoft.com/api/management/v2/restore_change/ \
+    -u username:password -d '{ "change_uid": 126 }'
+```
+
+Parameter | Description
+--------- | -----------
+`dataset_uid` <br> *string* | identifier of the dataset to restore to a previous change
+`change_uid` <br> *string* | identifier of the change to restore. This parameter must be sent in the json format, inside a json an object
```
Comment (meganelacheny) [inline source/includes/_changes.md]: "inside a json an object"? It does not makes sense to me.

File: source/includes/_changes.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+* a reference to the user who created the new state
+* the day and time at which the change occured
+* the difference between the old and the new state of the dataset
+* the sections in which the change happened
+
+### Attributes
+
+Attribute | Description
+--------- | -----------
+`change_uid` <br> *string* | unique identifier for the change
```
Comment (meganelacheny) [inline source/includes/_changes.md]: Also to ensure homogeneity in the documentation, could you please write the first letter of the first word of each description in capital letter?

**Review by meganelacheny — CHANGES_REQUESTED** [review]

File: source/includes/_changes.md (hunk 5 above)
Comment (Aquassaut) [reply source/includes/_changes.md]: :+1:

File: source/includes/_changes.md (hunk 8 above)
Comment (Aquassaut) [reply source/includes/_changes.md]: :+1:

File: source/includes/_changes.md (hunk 6 above)
Comment (Aquassaut) [reply source/includes/_changes.md]: :+1:

File: source/includes/_changes.md (hunk 7 above)
Comment (Aquassaut) [reply source/includes/_changes.md]: Should be "a json object" indeed :+1:

**Pull request merged**